checks.d/mongo.py

```diff
@@ -200,11 +243,20 @@ def check(self, instance):
         username = parsed.get('username')
         password = parsed.get('password')
         db_name = parsed.get('database')
-        clean_server_name = server.replace(password, "*"*5) if password is not None else server
+        clean_server_name = server.replace(password, "*" * 5) if password is not None else server
 
         tags = instance.get('tags', [])
         tags.append('server:%s' % clean_server_name)
 
+        # Get the list of metrics to collect
+        collect_tcmalloc_metrics = _is_affirmative(
+            instance.get('collect_tcmalloc_metrics', False)
```

Comment:
If we're going to add an option for collect_tcmalloc_metrics then we can use this to decide whether to request tcmalloc stats on older versions of MongoDB as well. They're starting to return them in 3.1.8 and later by default, but you can request them on older versions of MongoDB by passing `tcmalloc:true` when calling serverStats on line 316.

https://jira.mongodb.org/browse/SERVER-19941